umami.libraries.yml

```diff
@@ -5,6 +5,8 @@ global:
       css/base.css: {}
       css/components/blocks/branding/branding.css: {}
       css/components/blocks/search/search.css: {}
+      css/components/content-types/recipe/recipe.css: {}
```

Comment:
We have "components/content"folder in the dev branch. Should we have that and this too?